src/ui/server/controllers/insurance/policy/loss-payee-details/index.ts

```diff
@@ -61,3 +67,44 @@ export const get = (req: Request, res: Response) => {
     userName: getUserNameFromSession(req.session.user),
   });
 };
+
+/**
+ * post
+ * Run validation and either renders template with errors or redirects to next page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.redirect} Next part of the flow
+ */
+export const post = async (req: Request, res: Response) => {
```

Comment:
I don't think we need `async` or  try/catch at the bottom of this controller , there's no `await` calls

Reply:
I've kept this for the next PR with save which will need it